buyer-api/test/blockchain/contractEventSubscribers/dataOrderSubscribers.test.js

```diff
@@ -9,38 +9,36 @@ import {
 const it = test.serial;
 
 const data = { buyer: 'somebuyer', orderId: 1 };
+const tx = { transactionHash: 'somehash' };
 
 it('does not update the order status when the buyer param is not the current account', async (assert) => {
-  getAccount.returns({ address: 'someAccount' });
+  getAccount.resolves({ address: 'someAccount' });
   await onDataOrderClosed(data);
   assert.false(dataOrders.update.called);
 });
 
 it('updates the order status to closed', async (assert) => {
-  getAccount.returns({ address: 'somebuyer' });
-  fetchDataOrder.returns({ buyerUrl: '/orders/2/offchain-data' });
+  getAccount.resolves({ address: 'somebuyer' });
   await onDataOrderClosed(data);
   assert.is(dataOrders.update.firstCall.args[1].status, 'closed');
 });
 
 it('does not update the order when the buyer param is not the current account ', async (assert) => {
-  getAccount.returns({ address: 'someAccount' });
-  await onDataOrderCreated(data, { transactionHash: 'somehash' });
+  getAccount.resolves({ address: 'someAccount' });
+  await onDataOrderCreated(data, tx);
   assert.false(dataOrders.store.called);
 });
 
 it('does not update the order when the order is already closed', async (assert) => {
-  getAccount.returns({ address: 'somebuyer' });
-  fetchDataOrder.returns({ buyerUrl: '/orders/2/offchain-data' });
-  dataOrders.fetch.returns({ status: 'closed' });
-  await onDataOrderCreated(data, { transactionHash: 'somehash' });
+  getAccount.resolves({ address: 'somebuyer' });
+  dataOrders.fetch.resolves({ status: 'closed' });
+  await onDataOrderCreated(data, tx);
   assert.false(dataOrders.store.called);
 });
 
 it('stores the order', async (assert) => {
-  getAccount.returns({ address: 'somebuyer' });
-  fetchDataOrder.returns({ buyerUrl: '/orders/2/offchain-data' });
-  dataOrders.fetch.returns({ status: 'creating' });
-  await onDataOrderCreated(data, { transactionHash: 'somehash' });
+  getAccount.resolves({ address: 'somebuyer' });
+  dataOrders.fetch.resolves({ status: 'creating' });
+  await onDataOrderCreated(data, tx);
   assert.snapshot(dataOrders.store.firstCall.args, { id: 'dataOrders.store().args' });
```

Comment:
I think you're using `update` instead of `store` now, so `store.firstCall` will be `null`, and snapshot fails because of that.